packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -1,56 +1,55 @@
-import { html, ref } from "@microsoft/fast-element";
-import type { ViewTemplate } from "@microsoft/fast-element";
-import { endSlotTemplate, startSlotTemplate } from "../patterns/start-end.js";
-import type { FoundationElementTemplate } from "../foundation-element/foundation-element.js";
-import type { AccordionItem, AccordionItemOptions } from "./accordion-item.js";
+import { ElementViewTemplate, html, ref } from "@microsoft/fast-element";
+import { createEndSlotTemplate, createStartSlotTemplate } from "../patterns/start-end.js";
+import type { AccordionItemOptions, FoundationAccordionItem } from "./accordion-item.js";
 
 /**
- * The template for the {@link @microsoft/fast-foundation#(AccordionItem:class)} component.
+ * The template for the {@link @microsoft/fast-foundation#(FoundationAccordionItem:class)} component.
  * @public
  */
-export const accordionItemTemplate: FoundationElementTemplate<
-    ViewTemplate<AccordionItem>,
-    AccordionItemOptions
-> = (context, definition) => html`
-    <template class="${x => (x.expanded ? "expanded" : "")}">
-        <div
-            class="heading"
-            part="heading"
-            role="heading"
-            aria-level="${x => x.headinglevel}"
-        >
-            <button
-                class="button"
-                part="button"
-                ${ref("expandbutton")}
-                aria-expanded="${x => x.expanded}"
-                aria-controls="${x => x.id}-panel"
-                id="${x => x.id}"
-                @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
+export function createAccordionItemTemplate(
+    options: AccordionItemOptions = {}
+): ElementViewTemplate<FoundationAccordionItem> {
+    return html<FoundationAccordionItem>`
```

Comment:
Is there other code we'd expect to happen before returning the template? For instance, in some cases maybe there's some logic based on the options structure and values provided. I only noticed here because the indentation changed and GitHub didn't handle it well. I like the idea of following the same function definition for all templates, even ones like this without logic, but if we don't expect that perhaps we should default to the `const` style function definition.